changelog.txt

```diff
@@ -1,3 +1,7 @@
+v1.1.0 (Aug 20, 2020)
+=====================
+- Release (no changes since beta2)
```

Comment:
@tanema Wasn't sure whether move the beta/beta2 entries into here or leave as is, any thoughts?

Reply:
I think I would merge them into the final release just so it is clear what is in the release